src/index.html

```diff
@@ -1,13 +1,431 @@
+            </div>
+
+            <div class="testimonials__card">
+              <img
+              src="images/Testimonials/Michel.Grover.png"
+              alt="Tal Gilad"
+              class="testimonials__photo"
+              >
+
+              <p class="testimonials__text">AIR raises the agency bar to stratospheric
+                heights on both creative output and client service.
+              </p>
+              <p class="testimonials__person">Michel Grover</p>
+              <p class="testimonials__position">Hulu</p>
+            </div>
+          </article>
+        </section>
+
+        <section class="vision">
+          <article class="vision__container">
+            <h2 class="vision__title">Vision, Passion, Results</h2>
+            <p class="vision__paragraph">We are sure that first-rate job is possible only if all three components are united.</p>
+            <a href="#send-us-a-message" class="vision__button-container">
+              <button class="vision__button">Apply</button>
+            </a>
+          </article>
+        </section>
+
+        <div class="page__bottom">
+          <section class="send-us-a-message" id="send-us-a-message">
+            <article class="send-us-a-message__container">
+              <h2 class="send-us-a-message__title">Send us  a message</h2>
+
+              <form action="#"
+              name="sent-us"
+              class="send-us-a-message__form"
+              >
+
+                <label class="send-us-a-message__label">
+                  <input
+                  type="email"
+                  name="email"
+                  class="send-us-a-message__item"
+                  placeholder="Your email"
+                  required
+                  >
+                </label>
+
+                <label class="send-us-a-message__label">
+                  <input
+                  type="text"
+                  name="name"
+                  class="send-us-a-message__item"
+                  placeholder="Your name"
+                  required
+                  >
+                </label>
+
+                <label class="send-us-a-message__label">
+                  <textarea
+                  type="text"
+                  name="mesage"
+                  class="send-us-a-message__item send-us-a-message__item--size"
+                  placeholder="Your message"
+                  required
+                  ></textarea>
+                </label>
+
+                <button
+                type="submit"
+                class="send-us-a-message__button"
+                >
+                  Send
+                </button>
+              </form>
+            </article>
+          </section>
+
+          <section class="contact-us" id="contact-us">
+            <article class="contact-us__container">
+              <h2 class="contact-us__title">Contact us</h2>
+
+              <div class="contact-us__items">
+                <span class="contact-us__number">
+                  <a
```

Comment:
Please fix formatting

```suggestion
                  <a
                    class="contact-us__link"
                    href="tel:654 321 987"
                    target="_blank"
                  >
                    654 321 987
                  </a>
```